modules/react/tabs/stories/stories_testing.tsx

```diff
@@ -66,18 +73,58 @@ export const TabStates = withSnapshotsEnabled(() => (
   </>
 ));
 
-export const Bidirectionality = withSnapshotsEnabled(() => {
+export const BasicTabs = withSnapshotsEnabled(() => {
+  return <Basic />;
+});
+
+BasicTabs.parameters.chromatic.viewports = [480, 1200];
```

Comment:
I don't understand the purpose of changing the viewport here. This is the basic example which does not overflow, so the 480px width version simply gets clipped in the screenshot. The `TabStates` story already captures what the `Basic` example does. What does the `BasicTabs` story add for visual regression testing? We pay per Story captured, so each story should add something useful.